app/domain/validation.py

```diff
@@ -68,6 +69,7 @@ def validate_mission(
     employee_version_end_time=None,
     is_auto_validation=False,
     is_admin_validation=None,
+    justification=None,
```

Comment:
question : dans la méthode au dessus (`pre_check_validate_mission_by_admin`), le `# Special case #2 - last activity started more than 24h ago and still running` ne pourra plus se produire, si ?

Reply:
je pense que le cas peut se produire car cela regarde seulement s'il y a 24h écoulé depuis le début de l'activité, là où les auto validations s'effectuent 1 jour ouvré après la première activité reçue.
De plus, en théorie les auto validations s'appliquent 1 jour ouvré après la première activité reçue, mais en pratique pas forcément (il faut que le job runne).
Dans un cas on regarde le timestamp de début d'activité versus l'heure de réception dans l'autre aussi.
Pour toutes ces raisons je pense que c'est mieux de laisser l'ancien comportement tel quel ?